Resources/Prototypes/Catalog/VendingMachines/Inventories/theater.yml

```diff
@@ -41,6 +41,8 @@
     ClothingUniformJumpsuitKimono: 1
     ClothingHeadHatCasa: 1
     ClothingHeadHatHairflower: 1
+    ClothingHeadHatGladiator: 1
+    ClothingUniformJumpsuitGladiator: 1
```

Comment:
if these are meant to be armor at some point, then they should be taken out of the theater vendor and just used properly in the future

Reply:
it isn't armor. It both are just uniform/jumpsuit and head/hat. Also both of them were in theatre vendor in SS13 (on paradise).
I hope that solves the question.